atlasdb-backup/src/main/java/com/palantir/atlasdb/backup/AtlasBackupService.java

```diff
@@ -91,6 +89,7 @@ public static AtlasBackupService create(
 
     private void storeBackupToken(InProgressBackupToken backupToken) {
         inProgressBackups.put(backupToken.getNamespace(), backupToken);
+        backupPersister.storeImmutableTimestamp(backupToken);
```

Comment:
Let's add a todo to eventually stop doing this. We want it now for backwards compatibility, but if there is not a valid use case for it (maybe it's used to repair targeted sweep tables, haven't checked?), we should get rid of it when reasonable.